release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,21 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-column-management"]
+==== Column management on resources page
+
+A *Manage columns* icon image:manage-columns.png[title="Manage Columns icon"] is
+now added to some resources pages, for example the Pods page. When you click on
+the icon, default column names are listed with check boxes on the left side of
+the modal and additional column names are listed on the right. Deselecting a
+check box will remove that column from the table view. Selecting a check box
+will add that column to the table view. A maximum combination of nine columns
+from both sides of the modal are available for display at one time. Clicking
+*Save* will save the changes that you make. Clicking *Restore default columns*
```

Comment:
It should be `Restore Default Columns` with the first character for every word capitalized